transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -469,7 +455,30 @@ public void connect(SocketAddress remoteAddress, SocketAddress localAddress, Cha
         }
     }
 
-    private final class LastInboundHandler extends ChannelInboundHandlerAdapter {
+    private static final class LastInboundHandler extends ChannelInboundHandlerAdapter {
+        Queue<Object> inboundMessages;
+        Throwable lastException;
+        /**
+         * Returns the {@link Queue} which holds all the {@link Object}s that were received by this {@link Channel}.
+         */
+        Queue<Object> inboundMessages() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename method "inboundMessages" to prevent any misunderstanding/clash with field "inboundMessages" defined on line 459. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1845)